org-roam.el

```diff
@@ -415,10 +415,13 @@ This is equivalent to removing the node from the graph."
                          (insert (format "** [[file:%s][%s]]\n"
                                          file-from
                                          (org-roam--get-title-or-slug file-from)))
-                         (dolist (content contents)
```

Comment:
we can:
1. propertize the content with the file and start-point.
2. create a sparse-keymap containing a binding for `RET` to `org-roam--jump-to-point`, using the text properties (file and point) to jump to the actual point.

```
     (let ((map (make-sparse-keymap)))
       (define-key map [mouse-2] 'operate-this-button)
       (put-text-property link-start link-end 'keymap map))
```

See https://www.gnu.org/software/emacs/manual/html_node/elisp/Clickable-Text.html